client/blocks/signup-form/index.jsx

```diff
@@ -1136,6 +1137,24 @@ class SignupForm extends Component {
 			: formState.getFieldValue( this.state.form, 'email' );
 	};
 
+	handleCreateAccountError = ( error, email ) => {
+		if ( this.props.handleCreateAccountError ) {
+			return this.props.handleCreateAccountError( error, email );
+		}
+
+		if ( [ 'already_taken', 'already_active', 'email_exists' ].includes( error.error ) ) {
```

Comment:
I noticed we have this check in a couple of spots. Should we extract it somewhere for reuse?

`isExistingAccountError = (error) => [ 'already_taken', 'already_active', 'email_exists' ].includes( error.error )`

Reply:
Also agreed making this change as well

Reply:
[9179d12](https://github.com/Automattic/wp-calypso/pull/92013/commits/9179d12d962c0cd58aabb956012cc961e23a1007)